daemon/oci_linux_test.go

```diff
@@ -50,6 +50,69 @@ func TestTmpfsDevShmNoDupMount(t *testing.T) {
 	assert.Check(t, err)
 }
 
+// TestTmpfsExecOptions tests both default of "noexec" and
+// possibility to override it using "exec" option.
+func TestTmpfsExecOptions(t *testing.T) {
```

Comment:
I don't see how this test case brings some additional value. Checking that exec and noexec can be used and result in mounts with proper options is already done in a previous commit. I would drop this one.

Reply:
Thanks, removed. I had added it to test for the default value of `noexec` set by the daemon on tmpfs volumes, but that default value is not introduced by this PR anyway.